### PR TITLE
Replaced deprecated functions with suggestions in ImageNet example.

### PR DESCRIPTION
**Pull request by OMGtechy**
I wanted to have a look at an ImageNet example and came across this. However, it was full of deprecation warnings, so I've dropped in the suggested functions in their stead. There's still one warning left, however:

```
2019-11-29 00:50:28.988072: W tensorflow/core/framework/op_def_util.cc:370] Op BatchNormWithGlobalNormalization is deprecated. It will cease to work in GraphDef 
version 9. Use tf.nn.batch_normalization().
```

I wasn't as sure as to how to respond to this one, as there's no mention of BatchNormWithGlobalNormalization anywhere in the example, so I've left that one.

Regardless, this set of changes I've made is at least an improvement. Now both 2.0 and 1.15 users can use this example; beforehand only 1.15 users could.

I have signed the contributors agreement.

### CODE REVIEW

**Comment by jaeyounkim** [comment]
I am closing this PR since the tutorials directory is deprecated.
